assets/src/edit-story/components/form/textArea.js

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useCallback, useMemo } from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { rgba } from 'polished';
+
+const StyledTextArea = styled.textarea`
+  width: 100%;
+  padding: 0;
+  border: none;
+  appearance: none;
+  box-shadow: none !important;
+  outline: none;
+  background-color: transparent;
+  color: ${({ theme }) => theme.colors.fg.white};
+  font-family: ${({ theme }) => theme.fonts.body2.family};
+  font-size: ${({ theme }) => theme.fonts.body2.size};
+  line-height: ${({ theme }) => theme.fonts.body2.lineHeight};
+  letter-spacing: ${({ theme }) => theme.fonts.body2.letterSpacing};
+  resize: ${({ resizeable }) => (resizeable ? 'auto' : 'none')};
+
+  &:disabled {
+    background-color: transparent;
+    color: ${({ theme }) => theme.colors.fg.white};
+  }
+`;
+
+const Container = styled.div`
+  width: 100%;
+  color: ${({ theme }) => rgba(theme.colors.fg.white, 0.3)};
+  background-color: ${({ theme }) => rgba(theme.colors.bg.black, 0.3)};
+
+  ${({ disabled, readOnly }) => (disabled || readOnly) && `opacity: 0.3`};
+
+  padding: 6px 12px;
+  padding-right: 6px;
```

Comment:
This is just `padding: 6px 6px 6px 12px;` or `padding: 6px; padding-left: 12px;`.